tests/angular/custom-ignore-attributes/index.spec.ts

```diff
@@ -18,7 +18,7 @@ describe('MyComponent', () => {
     it('should have a div', async () => {
         await fixture.whenStable();
 
-        expect(hostComponent.div).toBeDefined();
-        expect(hostComponent.myMethod).not.toHaveBeenCalled();
+        await expect(hostComponent.div).toBeDefined();
+        await expect(hostComponent.myMethod).not.toHaveBeenCalled();
```

Comment:
expect isn't async right? There is a separate [expectAsync](https://jasmine.github.io/api/5.3/global.html#expectAsync)

Reply:
Good catch, I had just mindlessly added the await. There was a lot going on here:

1. this project was importing `@types/jasminewd2` to get the jasmine types, and its matchers (e.g. `toBeDefined()` were async (the whole reason we had disabled this rule for tests in the first place)
2. jasminewd2 is from the Protractor days and no longer a dependency of Angular but this repo and Skyline still (incorrectly) reference it in a few spots

The fix was to remove it and replace it with `@types/jasmine`, which has synchronous matchers. This allows the tests to pass linting without the `await`.

While doing this I noticed that the playwright tests were somehow relying on the jasmine definition of `describe` so I updated one of them to use `test.describe` which is how Playwright does this.

I will make Skyline PRs to remove jasminewd2 as well.

Reply:
https://dev.azure.com/ni/DevCentral/_git/Skyline/pullrequest/801169